cmd/loopd/swapclient_server.go

```diff
@@ -266,10 +267,13 @@ func (s *swapClientServer) LoopOutQuote(ctx context.Context,
 		return nil, err
 	}
 
+	dest := hex.EncodeToString(quote.SwapPaymentDest[:])
 	return &looprpc.QuoteResponse{
-		MinerFee:  int64(quote.MinerFee),
-		PrepayAmt: int64(quote.PrepayAmount),
-		SwapFee:   int64(quote.SwapFee),
+		MinerFee:        int64(quote.MinerFee),
+		PrepayAmt:       int64(quote.PrepayAmount),
+		SwapFee:         int64(quote.SwapFee),
+		SwapPaymentDest: dest,
```

Comment:
Why is this actually a string field?

Reply:
It was from before, do you think it makes more sense as raw bytes?

Reply:
Now that we move it, I think there is an opportunity to change. Idk why it was string based, but in general I don't think we should do hex strings. The only reason I know of is for parameters that show up in rest urls.

Reply:
Changed to `bytes`